src/pom.xml

```diff
@@ -468,6 +468,12 @@
     <artifactId>gt-process-feature</artifactId>
     <version>${gt.version}</version>
    </dependency>
+   <dependency>
+    <groupId>org.geotools</groupId>
+    <artifactId>gt-geopkg</artifactId>
+    <version>${gt.version}</version>
+   </dependency>
+
```

Comment:
I had to add this to the gs-main pom.xml as well, or some of the extension modules fail tests.

Reply:
The breakage is actually WCS 1.0, and its because we end up with "GeoPackage (mosaic)" in the supported formats: `<wcs:formats>GeoPackage (mosaic)</wcs:formats>`.